packages/front-end/components/Features/ExperimentSummary.tsx

```diff
@@ -19,20 +20,26 @@ const percentFormatter = new Intl.NumberFormat(undefined, {
 
 export default function ExperimentSummary({
   rule,
-  experiment,
   feature,
 }: {
   feature: FeatureInterface;
-  experiment?: ExperimentInterfaceStringDates;
```

Comment:
When we render the feature page, we look in the rules to find any experiment ids to link to, but we don't search through the draft rules as well.  A better fix for this bug is to look in the draft rules too.

So revert changes to this file and make the change on the back-end instead

Reply:
Thanks @jdorn  would update this